### PR TITLE
Bug 1678090: Add docs for the new Mozilla Phabricator Emails

### PR DESCRIPTION
**Pull request by mitchhentges**
Briefly explains what the new emails are and how to opt-in.

### CODE REVIEW

File: phabricator-user.rst [hunk 1]
```diff
@@ -358,6 +358,31 @@ We will, however, display some revision metadata in associated
...
 <https://bugzilla.mozilla.org/show_bug.cgi?id=1489706>`_.
 
+Mozilla Phabricator Emails
+==========================
+
+.. image:: images/phab-email-new.png
+   :align: center
+   :alt: Screenshot of an example new Phabricator email
+
+There's a new Phabricator email system current in development
```
Comment (cgsheeh) [inline phabricator-user.rst:368]: `currently`

File: phabricator-user.rst [hunk 2]
```diff
@@ -358,6 +358,31 @@ We will, however, display some revision metadata in associated
...
+with the following improvements over the existing system:
+
+* There's less noise per email.
```
Comment (cgsheeh) [inline phabricator-user.rst:371]: I'd probably expand on point 1 slightly and make point 3 slightly more formal. Suggestion:

* There's less noise per email, and fewer emails sent per event.
* Relevant contextual information is now more clearly available.
* They're faster to read and have less visual clutter than stock Phabricator emails.

**Review by cgsheeh — CHANGES_REQUESTED** [review]
Fix typo and take suggestions at your discretion. :)

File: phabricator-user.rst (hunk 2 above)
Comment (mitchhentges) [reply phabricator-user.rst:371]: > They're faster to read and have less visual clutter than stock Phabricator emails.

IMHO, this feels like a duplicate of the first point: "there's less noise".
How about something like: "They're visually structured to be helpful and fast to read"?

Comment (cgsheeh) [reply phabricator-user.rst:371]: > How about something like: "They're visually structured to be helpful and fast to read"?

I like it :+1:

**Review by cgsheeh — APPROVED** [review]

**Pull request merged**